spec/features/user_sees_dashboard_functionality_when_signed_in_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'As a logged in user' do
+  before :each do
```

Comment:
Almost redundant use of mock oauth signing in but used to show that the rest of the page only shows up with a successful login.